Spanner/src/Connection/Grpc.php

```diff
@@ -275,6 +294,74 @@ public function getInstanceConfig(array $args)
         ]);
     }
 
+    /**
+     * @param array $args
+     */
+    public function createInstanceConfig(array $args)
+    {
+        $instanceConfigName = $args['name'];
+
+        $instanceConfig = $this->instanceConfigObject($args, true);
+        $res = $this->send([$this->getInstanceAdminClient(), 'createInstanceConfig'], [
+            $this->pluck('projectName', $args),
+            $this->pluck('instanceConfigId', $args),
+            $instanceConfig,
+            $this->addResourcePrefixHeader($args, $instanceConfigName)
+        ]);
+
+        return $this->operationToArray($res, $this->serializer, $this->lroResponseMappers);
+    }
+
+    /**
+     * @param array $args
+     */
+    public function updateInstanceConfig(array $args)
+    {
+        $instanceConfigName = $args['name'];
+
+        $instanceConfigArray = $this->instanceConfigArray($args);
+
+        $fieldMask = $this->fieldMask($instanceConfigArray);
```

Comment:
From what I found in local testing, I think the `name` key needs to be removed before sending it to the `fieldMask` method.
Let me know if you think that's not the case.

Reply:
You're right. I've changed that.

Are there any plans for running the integration tests on real Cloud Spanner for PRs in this repository? Or is there any way that they can be kicked off manually? I guess that would have caught this earlier, as it is probably a case of making a small change after having executed all the integration tests, and then forgetting to run them all once more after this change.

Reply:
The integration tests run as a continuous job but not on PRs.

You can test them manually by running `phpunit -c phpunit-system.xml.dist --verbose tests/System/AdminTest`.

Optionally you could use the `--filter` argument if you don't want other tests to run. It accepts a regex.